android/src/main/java/com/tailscale/ipn/ui/util/AppVersion.kt

```diff
@@ -0,0 +1,17 @@
+package com.tailscale.ipn.ui.util
```

Comment:
// Copyright (c) Tailscale Inc & AUTHORS
// SPDX-License-Identifier: BSD-3-Clause

Reply:
We should find a way to make this a local check so it will prevent you from building. I always forget it 😅

Reply:
if you will make a github issue for it, I'll create the gradle task checking for the heading :)